WordPressKit/HTTPRequestBuilder.swift

```diff
@@ -56,6 +56,29 @@ final class HTTPRequestBuilder {
         append(query: [URLQueryItem(name: name, value: value)], override: override)
     }
 
+    func query(_ dict: [String: Any]) -> Self {
+        dict.reduce(self) {
+            $0.query(name: $1.key, value: $1.value)
+        }
+    }
+
+    func query(name: String, value: Any) -> Self {
```

Comment:
How do you feel about

```suggestion
    func queryParameter(name: String, value: Any) -> Self {
```

?

I don't know why (which is in itself a reason to discard my suggestion) but simply "query" for a method that sets a parameter only seems unclear.

Reply:
I feel like these APIs look less strange from call sites. Here is an example:

```
HTTPRequestBuilder(url: "https://public-api.wordpress.com")
    .method(.post)
    .append(path: "/rest/v1.2/sites/\(id)/posts")
    .query(name: "context", value: "mobile")
    .query(name: "mode", value: "html")
```